core/components/header/index.tsx

```diff
@@ -1,47 +1,53 @@
 import { ShoppingCart, User } from 'lucide-react';
-import { NextIntlClientProvider } from 'next-intl';
-import { getLocale, getMessages } from 'next-intl/server';
+import { getLocale } from 'next-intl/server';
 import { ReactNode, Suspense } from 'react';
 
 import { getSessionCustomerId } from '~/auth';
 import { FragmentOf, graphql } from '~/client/graphql';
+import { logoTransformer } from '~/data-transformers/logo-transformer';
+import { localeLanguageRegionMap } from '~/i18n';
 
+import { headerLinksTransformer } from '../../data-transformers/header-links-transformer';
```

Comment:
🍹 consistency with logo transformer import above
```suggestion
import { headerLinksTransformer } from '~/data-transformers/header-links-transformer';
```